app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+from models import Actor
```

Comment:
different types of imports should be separate wiht blank line and grouped accordingly